cmd/kube-addon-manager/README.md

```diff
@@ -26,37 +26,3 @@ Otherwise it will be omitted.
 #### Images
 
 addon-manager images are pushed to `k8s.gcr.io`. As addon-manager is built for multiple architectures, there is an image per architecture in the format - `k8s.gcr.io/kube-addon-manager-$(ARCH):$(VERSION)`.
-
-#### How to release
```

Comment:
I was hoping to get some suggestions. There's a number of different methods leveraged across the different components, and haven't gotten a good answer as to the state-of-the-art in that regard

Reply:
it should not be part of the release because the underlying mechanism was deprecated a long time ago.
the readme should reflect that.

new efforts around cluster addons are happening in kubernetes-sigs/cluster-addons

Reply:
I don't think cluster lifecycle can both disown a project and refuse changes to it ...

Reply:
This shouldn't be part of the release build because we shouldn't need to re-build it on every commit / isn't tied to the kubernetes release lifecylce, as opposed to say, kube-scheduler, and it won't be bundled with the core release artifacts. Other images like this are the debian base images, go-runner, e2e images ...

There are other default images in the release build that are borderline inappropriate as-is (like the image consumed by sonobuoy), we should not extend that.

As for how you should release an image like this, we have automatic post-submit image builds used for e.g. the images under e2e, etc. a rabbit hole to go down:
https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io
https://github.com/kubernetes/test-infra/tree/master/images/builder

Basically we automatically run a cloudbuild.yaml in post-submit, and then you can file a promotion from the staging registry to the prod registry in the k8s.io repo. Anything in the project can release there at any time.

Reply:
> I don't think cluster lifecycle can both disown a project and refuse changes to it ...

it's not really a case where someone is "refusing" changes to this though.
all is recommendations.

that is why i've said:
> a couple of my comments for approvers to consider:

here:
https://github.com/kubernetes/kubernetes/pull/92978#pullrequestreview-465001212